src/platform/platformProtocol.ts

```diff
@@ -90,7 +90,7 @@ export interface PlatformProtocol {
   form: (
     definition: FormDefinition,
     controller: AbortController,
-    context: { componentId: UUID; modId?: RegistryId },
+    componentRef: ModComponentRef,
```

Comment:
```suggestion
    modComponentRef: ModComponentRef,
```

NIT: I know the platform is agnostic of React, but I could see `componentRef` being confused with React refs at a glance

Reply:
Agree - looks like I had missed this one